pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -479,12 +654,12 @@ You are now almost ready to run the Python bot. The simplest way to do so is wit
 </div>
 <br>
 
-In your `config.yml` file:
+In your `.env.server` file:
 
-* Set `urls.site` to `"web:8000"`.
+* Set `urls_site` to `"web:8000"`.
```

Comment:
```suggestion
* Set `urls_site` to `"web:8000/api"`.
```

Reply:
wouldn't this be `urls_site_api`?

Reply:
It's actually `urls_site` in the previous config scheme & that's because we defined a variable called 
`site` which takes that value, and then we use it to automatically compose the site_api & a bunche of other stuff
But yes, it's not the case anymore & should change

Reply:
This will need to be mentioned in the local run section as well